APsystemsEZ1/__init__.py

```diff
@@ -43,6 +43,12 @@ class APsystemsEZ1M:
     power status, alarm information, device information, and power limits.
     """
 
+    @dataclass
+    class DebounceVal:
```

Comment:
```suggestion
    class _DebounceVal:
```

To make it "private"